lib/storage/file.js

```diff
@@ -932,22 +932,29 @@ File.prototype.getMetadata = function(callback) {
  * @param {object} callback.policy - The document policy.
  *
  * @example
- * file.getSignedPolicy({
+ * var TWO_WEEKS_FROM_NOW = Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14);
```

Comment:
I'd be in favor of simply accepting anything that the Date object accepts, this is already a breaking change so users will have to update their code anyways.